src/sentry/replays/endpoints/organization_replay_count.py

```diff
@@ -44,15 +44,15 @@ def get(self, request: Request, organization: Organization) -> Response:
             return Response(status=404)
 
         try:
-            snuba_params, params = self.get_snuba_dataclass(
+            snuba_params, _ = self.get_snuba_dataclass(
```

Comment:
What's contained within `params` and why do we no longer target it?

Reply:
params is a dictionary, while SnubaParams is a dataclass https://github.com/getsentry/sentry/blob/531c61bbd3e01bc22e8840e41fc9b87c9239b059/src/sentry/search/events/types.py#L49. They contain the same data. For some reason, the QueryBuilder requires that you pass in both, but it will use the SnubaParams correctly. That probably needs to be refactored in their library. Can do so in a follow up.